package.json

```diff
@@ -45,7 +45,7 @@
   },
   "scripts": {
     "lint": "eslint --plugin markdown --ext js,md .",
-    "test": "mocha --require test/support/env --reporter spec --check-leaks --bail test/",
+    "test": "mocha --require test/support/env --reporter spec --check-leaks --global __core-js_shared__ --bail test/",
```

Comment:
We really shouldn't be adding a global leak with our changes. Is there a way to fix the leak? Just adding it to ignore is not really the best solution.

Reply:
@dougwilson It's from one of the libs used by either one of our existing dependencies, or mocha itself. Most probably by mocha itself.
It's a global leak introduced by `json5` and by `loadash` in the old versions that we are referencing. It's for deduping the code-js libs.
We are not using these in our code, so I don't think we can do anything about it - except increase dependency versions, which will break old node support.

Reply:
Interesting. Why doesn't the leak occur on `master` branch or other PRs, if that is the case?

Reply:
Even in this PR, it happened on and off. Sometimes by changing a single line with no interesting sideeffects.
I don't know if you recall but we've added and removed this before.
Also this happens only on specific very old versions of node, where that core-js dependency was required.

Reply:
Hm, strange. So that could just mean that these changes are causing it to be intermittent, not that it is intermittent without these changes. Of all the PRs and commits I push here, it has never happened, which is why it seems to point to the changes here somehow causing them to occur. It may help if maybe we approach this differently: can you just explain why it happens? For example, you say "happens only on specific very old versions of node, where that core-js dependency was required". So, which versions is that? Can we only add the exception to only those specific versions, then? You also say "except increase dependency versions". Increase which dependencies to which versions? If these are just test dependencies, we can change any of those to be whatever version is necessary for each different version of Node.js in our Travis CI file, like we are doing today.

Reply:
I did not deep dive that much. I only recall it was all the 0.x versions except <=0.8.
I don't really want or care to start eliminating it line by line, as those node versions are not supported for years now, and I can't do these tests without pushing little changes again and again to your CI and wait for the results. You could look at the previous CI results and see what I mean. Or maybe the whole history of CI on this branch.

What I can say for sure is that this even happened before switching between `'brotli' in process.versions` and `'createBrotliDecompress' in zlib`, and stopped with this change.

Reply:
@dougwilson More info: the users of this corejs quirk are `babel` (by requiring `json5`), and `eslint`.
`eslint` is the immediate suspect, as the modules it requires vary depending on the source code. An syntax combination somewhere could trigger another rule to be required, and then this leak will happen as a few of the rules require `lodash` which is the culprit.

Reply:
I agree fully that we should not add this. Since we will be updating most of the called out libs for 2.0 I think we can remove this and if it comes back we can address it separately from the rest of the changes here.

Reply:
I removed this.